components/landing/TheQiskitCapabilitiesSection/CodeCell.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <code class="code-cell">
+    <div class="code-cell__block" :class="{'code-cell__block_active' : activeCells[0]}">
+      <pre class="code-cell__line"><span class="code-cell__comment"># Qiskit quantum circuits libraries</span></pre>
```

Comment:
i like the `cell__line` naming

Reply:
As far as I can see, we are not really using the class `code-cell__line`, so we should remove it.